src/Tgstation.Server.Host/Components/Chat/Providers/IrcProvider.cs

```diff
@@ -252,9 +257,34 @@ await Task.Factory.StartNew(
 							.Select(x => x.TestMerge)
 							.Select(x =>
 							{
+								var status = string.Empty;
+								if (!previousRevisionInformation.ActiveTestMerges.Any(y => y.TestMerge.Number == x.Number))
+								{
+									status = "Added";
+								}
+								else if (revisionInformation.ActiveTestMerges!.Any(y => y.TestMerge.Number == x.Number && y.TestMerge.TargetCommitSha != x.TargetCommitSha))
+								{
+									status = "Updated";
+								}
+
 								var result = String.Format(CultureInfo.InvariantCulture, "#{0} at {1}", x.Number, x.TargetCommitSha![..7]);
+
 								if (x.Comment != null)
-									result += String.Format(CultureInfo.InvariantCulture, " ({0})", x.Comment);
+								{
+									if (!string.IsNullOrEmpty(status))
+									{
+										result += String.Format(CultureInfo.InvariantCulture, " ({1} - {0})", x.Comment, status);
```

Comment:
This is newer to the codebase. Prefer to use `$"{x}"` instead of `String.Format("{0}", x)`